SpatialGDK/Source/SpatialView/Private/SpatialSomething.cpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+#include "SpatialSomething.h"
+
+namespace SpatialView 
+{
+
+SpatialSomething::SpatialSomething(TUniquePtr<AbstractConnectionHandler> ConnectionHandler)
+: ConnectionHandler(MoveTemp(ConnectionHandler))
+{
+}
+
+void SpatialSomething::Advance()
+{
+	ConnectionHandler->Advance();
+	const uint32 OpListCount = ConnectionHandler->GetOpListCount();
+	for (uint32 i = 0; i < OpListCount; ++i)
+	{
+		View.ProcessOpList(*ConnectionHandler->GetNextOpList());
+	}
+	View.Advance();
+}
+
+void SpatialSomething::FlushMessagesToSend()
+{
+	ConnectionHandler->SendMessages(View.FlushLocalChanges());
```

Comment:
It feels like "FlushLocalChanges" should actually be "GetMessagesToSend".

Reply:
I quite like flush as it's ensuring that all stored changes have been applied to the view and it will only give each result once.